crates/host/src/wasmbus/mod.rs

```diff
@@ -22,7 +22,7 @@ use std::process::Stdio;
 use std::sync::Arc;
 
 use anyhow::{anyhow, bail, ensure, Context as _};
-use async_nats::jetstream::kv;
+use async_nats::jetstream::{context, kv};
```

Comment:
nit; can we just use `jetstream::context` directly please? The `context` namespace is way too ambiguous

Once we refactor this file into multiple smaller chunks, this would make sense, but for now it's way too opaque

Reply:
imported as `JetstreamContext`